app/enterprise/changelog.md

```diff
@@ -3,6 +3,25 @@ title: Kong Gateway (Enterprise) Changelog
 no_search: true
 no_version: true
 ---
+## 2.3.3.1
+**Release Date** 2021/04/13
+
+### Features
+
+#### Enterprise
+- Now users can set a custom login message and classification banner (top and bottom) on the Kong Manager login page. This classification banner persists beyond the login page and can be configured in the `kong.config` file.
+
+### Fixes
+
+#### Core
+- Fixed an issue occurring when using upstreams for load balancing where Kong was attempting to resolve the upstream name instead of the hostname and failing, with the errors, "name resolution failed" and "dns server error: 3 name error". The following updates correct the issue:
```

Comment:
```suggestion
- Fixed an issue that occurred when using upstreams for load balancing, where Kong was attempting to resolve the upstream name instead of the hostname and failing with the errors `name resolution failed` and `dns server error: 3 name error`. The following updates correct the issue:
```